cicd/pipelines/configuring-security-context-for-pods.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: ASSEMBLY
+[id="configuring-security-context-for-pods"]
+= Configuring the security context for pods
+include::_attributes/common-attributes.adoc[]
+:context: configuring-security-context-for-pods
+
+toc::[]
+
+The default service account for pods that {pipeline-shortnames} starts is `pipeline`. The security context constraint (SCC) associated with the `pipeline` service account is `pipelines-scc`. The `pipelines-scc` SCC is based the `anyuid` SCC, with minor differences as defined in the following YAML specification:
+
+.Example `pipelines-scc.yaml` snippet
+[source,yaml,subs="attributes+"]
+----
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+...
```

Comment:
Because this is a YAML snippet, I think the ellipses should be commented out. This is from the [openshift-docs/contributing_to_docs/doc_guidelines.adoc](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc) file:

```
For snippets or sections of a file, use an ellipsis (…​) to show that the file continues before or after the quoted block. For YAML, use ('#…​') instead.

apiVersion: v1
kind: Pod
metadata:
  labels:
    test: liveness
# ...
```